lib/src/entrypoint.dart

```diff
@@ -455,7 +455,7 @@ To update `$lockFilePath` run `$topLevelProgram pub get`$suffix without
         if (precompile) {
           await precompileExecutables();
         } else {
-          await _deleteExecutableSnapshots(changed: result.changedPackages);
+          await _deleteExecutableSnapshots();
```

Comment:
I guess we are throwing away information here. But at the same time, I don't know how we would communicate to a different compilation process what dependency entries changed.